bitbots_localization/package.xml

```diff
@@ -24,14 +23,11 @@
   <depend>bitbots_blackboard</depend>
   <depend>visualization_msgs</depend>
   <depend>builtin_interfaces</depend>
-  <depend>nav2_map_server</depend>
+  <!--depend>nav2_map_server</depend-->
```

Comment:
The `nav2_map_server` is still needed to show the map in rviz.

Reply:
Yes, but it's not released to ros rolling and therefore has no rosdep key. That's why it's commented out and not removed. This way we can add it back later.